taskcluster/ci/config.yml

```diff
@@ -65,6 +50,16 @@ workers:
                 by-level:
                     "3": adhoc-3-signing
                     default: adhoc-t-signing
+        mac-signing:
+            provisioner: scriptworker-prov-v1
+            implementation: scriptworker-signing
+            os: macosx
+            # We don't need a separate worker type
+            # for level 1 because RelEng are the only
+            # people who can pick off signing requests.
+            # This means that there's no security benefit
+            # to separating the level 1 / level 3 workers.
+            worker-type: adhoc-3-mac-signing
```

Comment:
I don't love this -- giving level 3 scopes to staging isn't great since releng can then bypass review, and we lose part of the security benefit of adhoc.
Did this work as-is on adhoc staging?

I'm somewhat leaning towards setting up the 2nd daemon on mac-v3-signing2 now.
- set up a level 1 clientId
- create a `/builds/dep-adhoc` or similar, set up the scriptworker.yaml and script_config.yaml accordingly
- sign with the dep key and don't notarize
- don't verify cot signatures

The main issue with this is we can't test notarization or "real" mac signing. The plus sides include having the dep worker more available for staging without having to ssh in, and not having the footgun of having to edit the config files when we switch the worker in between repos.

What do you think?

Reply:
Just to be clear: the comment above refers to signing in the production repo with level 1/dep & level 3/release -- not using the staging repo. (I realize this may be a confusing because I _did_ use the release certs in my tests against staging, but that was because I wanted to test notarization, too.)

After rereading the code, and your comments, I think I may have misunderstood the point of the `by-level` blocks in this file. When writing this patch, I thought it made it possible to use level 1 or level 3 workers in from the production adhoc-signing repo. Now, I think I'm wrong about that, and that production is always levels 3 and staging is always level 1. If this is the case, I'm 100% onboard with a separate daemon (and since level 1 will never notarize, I don't think we have any concurrency issues to worry about?).

> Did this work as-is on adhoc staging?

No :). But only because the worker type didn't match what we put in ci-config.

Reply:
That's correct, it's by the repo level, so staging is always level 1 and prod is always level 3.
I'm ok with this PR if we add the by-level back in here.
Setting the level 1 worker up can be a followup.